src/main/java/org/opensearch/timeseries/TimeSeriesAnalyticsPlugin.java

```diff
@@ -1672,6 +1679,17 @@ public List<NamedXContentRegistry.Entry> getNamedXContent() {
             );
     }
 
+    @Override
+    public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings settings) {
+        List<SystemIndexDescriptor> systemIndexDescriptors = new ArrayList<>();
+        systemIndexDescriptors.add(new SystemIndexDescriptor(ALL_AD_RESULTS_INDEX_PATTERN, "Time Series Analytics Results index pattern"));
+        systemIndexDescriptors
+            .add(new SystemIndexDescriptor(ALL_AD_DETECTOR_INDEX_PATTERN, "Time Series Analytics Detector index pattern"));
```

Comment:
Can you use CONFIG_INDEX instead of ALL_AD_DETECTOR_INDEX_PATTERN? We don't need ALL_AD_DETECTOR_INDEX_PATTERN.

Reply:
Can you change the description to "Time Series Analytics config index"?

Reply:
Changed